DTS-STN/Life-Journeys
# Adding organism for PageDetails, Adding DateModified Atom to include …

…in the page Details, fixed some linting, included unit tests and included the axe test framework

# Description

Adding the date modified and skeleton for the PageDetails.  included unit tests for the date modified.  included the Axe testing framework.  updated the environment variables in the configuration to default and format values accordingly.

1-3 sentence description of the feature you added.

## Test Instructions
run the app, look for the date modified information above the footer.  should be "DATE-IS-NA" in dev (unless .env variable is supplied), should be date of Team City build when deployed to cloud.

## Meets Definition of Done

- [ X] Meets design spec
- [ X] unit tests are included
